frontend/src/themes/components/alert.ts

```diff
@@ -0,0 +1,23 @@
+import { AlertProps } from "@chakra-ui/react";
+
+const Alert = {
+  baseStyle: (props: AlertProps) => {
+    const { status } = props;
+    if (status === "error") {
+      return {
+        container: {
+          background: "unset",
```

Comment:
`background: "unset"` here to make the `backgroundColor` work. This solution is described here: https://github.com/chakra-ui/chakra-ui/issues/6198